static/app/components/profiling/flamegraph/continuousFlamegraph.tsx

```diff
@@ -251,25 +258,31 @@ export function ContinuousFlamegraph(): ReactElement {
 
     for (const key in profileGroup.measurements) {
       if (key === 'cpu_energy_usage') {
-        measures.push({
-          ...profileGroup.measurements[key]!,
-          values: profileGroup.measurements[key]!.values.map(v => {
-            return {
-              elapsed_since_start_ns: v.elapsed_since_start_ns,
-              value: fromNanoJoulesToWatts(v.value, 0.1),
-            };
-          }),
-          // some versions of cocoa send byte so we need to correct it to watt
-          unit: 'watt',
-          name: 'CPU energy usage',
-        });
+        const measurements = profileGroup.measurements[key]!;
+        const values: ProfileSeriesMeasurement['values'] = [];
+
+        let offset = 0;
```

Comment:
This computes the value offset to the start of the profile

Reply:
Might be good to put into a helper function that takes a formatted for the value? I see it's copied for all the other charts below.